exporter/awskinesisexporter/README.md

```diff
@@ -15,6 +15,9 @@ The following settings can be optionally configured:
     - `kinesis_endpoint` (no default)
     - `region` (default = us-west-2): the region that the kinesis stream is deployed in
     - `role` (no default): The role to be used in order to send data to the kinesis stream
+- `encoding`
+    - `name` (default = jaeger): defines the export type to be used to send to kinesis (available is `otlp-proto`, `otlp-json`, `zipkin-proto`, `zipkin-json`, `jaeger`)
+    - `compression` (default = none): allows to set the compression type (defaults BestSpeed for all) before forwarding to kinesis (available is `flate`, `gzip`, `zlib` or `none`)
```

Comment:
It would be nice if we can separate out a PR for compression, it seems independent from the title of this PR to allow the configurable encodings

Reply:
I wanted to include compression here within the same PR due to the fact that using these new encodings can result in a record that is larger than what is permissible to write to kinesis.

But I do also agree it can be a seperate PR as a fast follow.